metricbeat/module/kubernetes/kubeproxy/README.md

```diff
@@ -0,0 +1,51 @@
+# Kubeproxy Stats
+
+## Version history
+
+- June 2019, `v1.13.0`
+
+## Resources
+
+- https://github.com/kubernetes/kubernetes/blob/master/pkg/proxy/metrics/metrics.go
+- https://kubernetes.io/docs/reference/command-line-tools-reference/kube-proxy/
+
+## Metrics insight
+
+process_cpu_seconds_total
```

Comment:
I understand these are the source metrics you are using, what's the exact meaning of this section?

Reply:
that section can be almost meaningless now, but further reviews might add here reasoning for including new metrics, deprecations ... any info that would help developing or testing, and that would save getting familiar with all metrics returned by prometheus at every release.

These readmes at metricsets are temptative, afaik there are only 2, so what is useful content here is very BD. If you consider this section won't be useful in the future or that it is not adding context, I'm ok removing it.